src/actions/profileAction.js

```diff
@@ -0,0 +1,19 @@
+/* @flow */
+import { SET_USERNAME, SET_USER_PHOTO } from "../constants/actionTypes";
+
+type SetUserNameAction = { type: "SET_USERNAME", userName: string };
+type SetUserPhotoAction = { type: "SET_USER_PHOTO", userPhoto: string };
```

Comment:
Move it to a separate file named types.js or smth like that, to avoid duplicates with https://github.com/KseniaKardash/itechart-react-native/pull/10/files#diff-38f0ba484516ccd2b9de60cef48ea3bdR14

Reply:
The same for:
https://github.com/KseniaKardash/itechart-react-native/pull/10/files#diff-2f965eb9693ae78a8763fa96d2af66e9R7
https://github.com/KseniaKardash/itechart-react-native/pull/10/files#diff-a3d21bc5c3bef010cdaab46d6fc99fe4R17
and any alike situation